src/Build/BackEnd/Shared/ConfigurationMetadata.cs

```diff
@@ -81,6 +93,11 @@ public ConfigurationMetadata(ITranslator translator)
         /// </summary>
         public string ToolsVersion => _toolsVersion;
 
+        public string PreviousPlatform { get; } = "";
```

Comment:
nit: defaulting to empty string here feels weird to me. Why not `null`?

Reply:
I used the empty string here because the string comparer  in InternalEquals throws an error if it is null.